extensions/extensions-docker/aissemble-model-training-api-containers/aissemble-model-training-api-sagemaker/pom.xml

```diff
@@ -36,14 +32,6 @@
                         </goals>
                         <configuration>
                             <wheelDependencies>
-                                <wheelDependency>
-                                    <artifactId>aissemble-foundation-core-python</artifactId>
-                                    <targetDirectory>${wheelOutputDirectory}</targetDirectory>
-                                </wheelDependency>
-                                <wheelDependency>
-                                    <artifactId>aissemble-foundation-pdp-client-python</artifactId>
-                                    <targetDirectory>${wheelOutputDirectory}</targetDirectory>
-                                </wheelDependency>
```

Comment:
Q: are these the transitive dependencies that you are removing?

Reply:
Some are transitive and some were just not used at all. The end result is we have on type `habushu` dependency which then contains the transitive ones.